hosted/aks/p0/p0_provisioning_test.go

```diff
@@ -52,10 +52,12 @@ var _ = Describe("P0Provisioning", func() {
 			Expect(err).To(BeNil())
 		})
 		AfterEach(func() {
-			err := helper.DeleteAKSHostCluster(cluster, ctx.RancherClient)
-			Expect(err).To(BeNil())
-			err = helper.DeleteAKSClusteronAzure(clusterName)
-			Expect(err).To(BeNil())
+			if ctx.ClusterCleanup {
+				err := helper.DeleteAKSHostCluster(cluster, ctx.RancherClient)
+				Expect(err).To(BeNil())
+				err = helper.DeleteAKSClusteronAzure(clusterName)
+				Expect(err).To(BeNil())
+			}
```

Comment:
Can you add a log info "Skipping cluster deletion" ?

Reply:
Done